### PR TITLE
[Draft] small bugfix in context menu

### PR DESCRIPTION
**Pull request by carlopav**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Draft/DraftTools.py [hunk 1]
```diff
@@ -4400,6 +4400,68 @@ def proceed(self,labelname):
                     except:
                         pass
 
+class AddToContainer():
```
Comment (vocx-fc) [inline src/Mod/Draft/DraftTools.py]: There is no parentheses in the class definition, `class AddToContainer:`

File: src/Mod/Draft/draftutils/utils.py [hunk 2]
```diff
@@ -573,28 +573,61 @@ def get_group_names():
...
+                or getType(obj) in ("Floor", "Building", "Site")):
+            containers.append(obj.Name)
+    return containers
+
 
 def ungroup(obj):
     """Remove the object from any group to which it belongs.
 
     A "group" is any object returned by `get_group_names`.
+    If group
```
Comment (vocx-fc) [inline src/Mod/Draft/draftutils/utils.py]: There's a missing comment here.

**Comment by carlopav** [comment]
My fault, the second commit wasn't supposed to be added to the PR. I'll drop it.

File: src/Mod/Draft/InitGui.py [hunk 3]
```diff
@@ -147,6 +147,7 @@ def Deactivated(self):
         Log("Draft workbench deactivated\n")
 
     def ContextMenu(self, recipient):
+        from DraftGui import translate
```
Comment (vocx-fc) [inline src/Mod/Draft/InitGui.py]: By the way, importing the `translate` function from `DraftGui` means that we have to process the entire `DraftGui` module. We don't really want to do this if we just want the `translate` function.

This is the reason I split the `translate` function into its own module, `draftutils.translate`, so we can import this component alone without pulling the rest of `DraftGui`. Perhaps we can test if this works correctly.

**Pull request merged**